docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policies.adoc

```diff
@@ -17,7 +17,7 @@ While defining the regular expression patterns consider the following parameters
 
 * You must ensure the `RegEx` pattern meets standard regular expression formatting standards for Python based regular expressions.
 * Criteria to consider when defining `RegEx` pattern.
-** Length of the resulting match must be between 5-100 characters.
+** Length of the resulting match must be between 5-100 characters and cannot be a match of all numbers and symbols without a single letter.
```

Comment:
The resulting match must be between 5 and 100 characters long and cannot consist solely of numbers and symbols without any letters.